mixer/adapter/denier/config/config.proto

```diff
@@ -27,7 +27,7 @@ option (gogoproto.gostring_all) = false;
 
 message Params {
 	// The error to return when denying a request.
-	google.rpc.Status status = 1 [(gogoproto.nullable) = false];
+	istio.mixer.v1.Status status = 1 [(gogoproto.nullable) = false];
```

Comment:
Won't this approach lead to issues when we pull in code that assumes google.rpc.Status? Say, like the gRPC libs?